Phonebook.Frontend/src/app/shared/dialogs/user-information/incorrect-user-information.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="pb-text-align-center">
+  <h2 mat-dialog-title i18n="User-InformationDialog|Headline@@Headline">Incorrect User Information</h2>
+  <div mat-dialog-content *ngIf="onMyProfile">
+    <span i18n="User-InformationDialog|Profile@@Your Profile">
+      Dear {{ data.firstname }},<br />
+      unfortunately, you can't change your user information via the Phonebook interface. <br />
+      If your data is incorrect, please contact your Human Resources department.
+    </span>
+  </div>
+  <div mat-dialog-content *ngIf="!onMyProfile">
+    <span i18n="User-InformationDialog|Profile@@Colleague Profile">
+      Dear {{ data.firstname }}<br />
```

Comment:
If the user-authentication is not working like in the preview the name here is the same name of the user u clicked.
E.g. : I am on Billy Abott's Profile the sentence is: "Dear Billy..."

Ether you remove the whole line (what is fine for me because it's not necessary) or you use person.firstname.
I also don't really know why you implemented this for you own. Because you can easy access the name like you did in `const dialogData`.